version-specific-2.9/app/components/focusing-outlet.js

```diff
@@ -0,0 +1,91 @@
+import Ember from 'ember';
+import getOwner from 'ember-getowner-polyfill';
+
+const { get } = Ember;
+
+let scrollLeft = 0;
+let scrollTop = 0;
+let handler = function(e) {
+  window.scrollTo(scrollLeft, scrollTop);
+  window.removeEventListener('scroll', handler);
+};
+
+let FocusingOutlet = Ember.Component.extend({
```

Comment:
I believe most (all?) of this is copy-pasta? Can we refactor this to not have two versions in the repo?